src/main/java/calculator/domain/CalculatorNumbers.java

```diff
@@ -0,0 +1,32 @@
+package calculator.domain;
+
+import java.util.List;
+import java.util.Objects;
+
+public class CalculatorNumbers {
+
+    private List<CalculatorNumber> calculatorNumberList;
```

Comment:
```suggestion
    private List<CalculatorNumber> calculatorNumbers;
```
List로 표현하기보단 복수형으로 표현하는게 일급 컬렉션에서는 더 자연스러울 것 같습니다.

Reply:
이후 클래스 네이밍시 고려하겠습니다!